CONTRIBUTING.md

```diff
@@ -0,0 +1,187 @@
+
+Format your commit messages following [Conventional Commits 1.0.0](https://www.conventionalcommits.org/en/v1.0.0/) (CC BY 3.0 netlify.com). Briefly:
+
+    <type>[(optional scope)][!]: <description>
+	[BLANK LINE]
+	[optional body]
+	[BLANK LINE]
+	[optional footer(s)]
+
+A typical commit message might look like:
+
+    docs: Add CONTRIBUTING manual
+	
+	Refs: #1234
+
+The description should summarize the changes in a single, short sentence. The description should be imperative, like the descriptions of the types below.
+
+The primary types are:
+- build: Change the build system, including tracking external dependencies
+- ci: Change the CI/CD system
+- docs: Update the package documentation
+- feat: Add a new feature
+- fix: Fix a bug or incorrect behavior
+- perf: Improve performance of a feature
+- refactor: Simplify maintainability without affecting performance or behavior
+- revert: Revert a previous commit
+- style: Correct a style mismatch (indentation, etc.)
+- test: Change or add to the test suite
+
+The optional scope must appear in parentheses and must consist of a noun describing the section of the code base that was changed.
+
+    docs(contrib): Update URLs in CONTRIBUTING manual
+
+The optional `!` must appear before the description whenever the commit introduces a breaking change - one that breaks compatibility with previous versions of the code base. Whenever the `!` marker appears, a BREAKING-CHANGE footer should also be included.
+
+The optional body may be included to provided additional information or context about the change. If it appears, it must be preceded by a blank line.
+
+The optional footers may be included to provide additional referential information and links to tracked issues, PRs, etc. Each footer must be preceded by a blank line. Footers must be formatted as `<footer-type>: <description>`, similar to the first line of commit messages. The description's format depends upon the footer-type. Known footer-types include:
+- Refs: A comma-separated list of commit hashes and/or issue and PR numbers. Issue and PR numbers must be prefixed with a `#`.
+- BREAKING-CHANGE: A description of the change that breaks compatibility with previous versions of the code, including the version number at which compatibility is broken. 
+
+A breaking change commit message might look like:
+
+    feat!: Switch to quantum scanner
+    
+    BREAKING-CHANGE: New scanner runs in constant time but doesn't support any existing hardware. 
+	Breaks compatibility with 1.0.0.
+
+Do not squash your commits unless requested by the maintainer. This aids discussion and revision during the review process.
+
+
+## Submitting PRs
+
+Before submitting a PR to westes/flex, test your changes either locally or using the GitHub Actions pipeline.
+
+### Testing locally
+
+1. Commit and/or stash your changes.
+1. Clean your working copy using
+
+    `git clean -xdf`
+
+1. Bootstrap the autotools using
```

Comment:
Should this section simply refer to the INSTALL doc -- so that we're not duplicating instructions?